hkmc2/jvm/src/test/scala/hkmc2/CompileTestRunner.scala

```diff
@@ -77,7 +69,20 @@ end CompileTestRunner
 
 object CompileTestRunner:
   
-  given cctx: CompilerCtx = CompilerCtx.fresh(io.FileSystem.default)
+  val pwd = os.pwd
+  val workingDir = pwd
+
+  val mainTestDir = workingDir/"hkmc2"/"shared"/"src"/"test"
+  val stdPath = mainTestDir / "mlscript-compile"
+  
+  val compilerPaths = new MLsCompiler.Paths:
+    val preludeFile = mainTestDir / "mlscript" / "decls" / "Prelude.mls"
+    val runtimeFile = mainTestDir / "mlscript-compile" / "Runtime.mjs"
+    val termFile = mainTestDir / "mlscript-compile" / "Term.mjs"
+  
+  val nodeModulesPath = workingDir / "node_modules"
```

Comment:
Any reason to move all of these in the companion? Doing so means they can no longer be overridden by other users of the test suite. (There is just one example with `BenchTestState` for now.)